src/zcbenchmarks.cpp

```diff
@@ -204,7 +204,7 @@ double benchmark_verify_equihash()
     CBlockHeader genesis_header = genesis.GetBlockHeader();
     struct timeval tv_start;
     timer_start(tv_start);
-    CheckEquihashSolution(&genesis_header, params.GetConsensus());
+    assert(CheckEquihashSolution(&genesis_header, 1, params.GetConsensus()));
```

Comment:
Nit: shouldn't this be 0? I know it doesn't matter.

Reply:
I picked 1 because we can't make all network upgrades active at genesis via `-nuactive` config options without re-generating the genesis block; by checking with height 1, we can run the benchmarks in RPC mode with either validator.